galleries/examples/animation/simple_scatter.py

```diff
@@ -31,3 +31,9 @@ def animate(i):
 # ani.save('scatter.gif', writer=writer)
 
 plt.show()
+
+# %%
+#
+# .. tags::
+#    component: animation, plot-type: scatter
```

Comment:
```suggestion
#    component: animation, plot-type: scatter,
```

Reply:
Without the comma, it tries to link to `plot-type-scatter-purpose-reference`.

Reply:
Confusingly, #27567 is fine without the commas? Maybe there's some parallel bug in `sphinx-tags`.